luigi/static/visualiser/index.html

```diff
@@ -167,35 +167,53 @@ <h4 class="modal-title" id="disiableWorkerLabel">Disable worker?</h4>
             {{/is_disabled}}
                 <div class="box-header with-border">
                     <h3 class="box-title">{{name}}</h3>
-                    <div class="box-tools pull-right">
-                      {{#num_unread_rpc_messages}}
-                      <span class="label-unread-worker-messages">{{num_unread_rpc_messages}} unread message(s)</span>
-                      {{/num_unread_rpc_messages}}
-                      {{^is_disabled}}
-                      {{#workers}}
-                      <div class="btn-group">
-                        <button type="button" class="btn btn-sm btn-default dropdown-toggle btn-set-workers" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                          Workers: <span id="label-n-workers" data-worker="{{name}}">{{workers}}</span> <span class="caret"></span>
+                    <div class="pull-right">
+                      <span class="btn-group">
+                        <button type="button" class="btn btn-sm btn-default dropdown-toggle btn-set-worker-custom-disconnect-delay" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                          Custom disconnect delay: <span id="label-custom-disconnect-delay" data-worker="{{name}}">{{custom_disconnect_delay}}</span> <span class="caret"></span>
```

Comment:
thoughts on `custom disconnect delay` defaulting to the scheduler config, such that this value isn't ever null? (assuming this is the null value you mention was displaying as an empty string)

Reply:
You lost me at 
> this configuration should be a scheduler rpc update.

However, i found a reasonably elegant way to save the frontend from dealing with the possible Null value. `worker_list` can obtain the current value. 

I don't see a better way right now. I'm not sure if there's a good way to make Scheduler construct Workers with a configuration value. Also, it seems that it would be complicated to make Worker weakref its Scheduler (to obtain scheduler config in, say, a property getter). 

https://github.com/koo5/luigi/commit/4c220d67728bd9ed91917d08c327dec37cc0584e

Also, nevermind about using `Worker.info`, as that seems to be for passing information from workers to scheduler.